src/trackers/HUNO.py

```
@@ -160,8 +152,22 @@ async def get_internal(self, meta: dict[str, Any]) -> dict[str, str]:
+        files = await super().get_additional_files(meta)
+
+        image_list = meta['HUNO_images_key'] if 'HUNO_images_key' in meta else meta['image_list']
+        description = await DescriptionBuilder(self.tracker, self.config).unit3d_edit_desc(
+            meta, image_list=image_list, approved_image_hosts=self.approved_image_hosts
+        )
+        files['description'] = ('description.txt', description.encode('utf-8'), 'text/plain')
+
+        if meta['bdinfo'] is not None:
+            mediainfo = await self.common.get_bdmv_mediainfo(meta, remove=['File size', 'Overall bit rate'])
+        else:
+            async with aiofiles.open(f"{meta['base_dir']}/tmp/{meta['uuid']}/MEDIAINFO_CLEANPATH.txt", encoding='utf-8') as f:
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)